Dockerfile.ocp

```diff
@@ -8,12 +8,17 @@ RUN prepare-efi.sh redhat
 
 FROM registry.ci.openshift.org/ocp/builder:rhel-9-base-openshift-4.16
 
+ENV REMOTE_SOURCES=${REMOTE_SOURCES:-.}
+ENV REMOTE_SOURCES_DIR=${REMOTE_SOURCES_DIR:-.}
 ENV PKGS_LIST=main-packages-list.ocp
 ARG EXTRA_PKGS_LIST
 
 COPY ${PKGS_LIST} ${EXTRA_PKGS_LIST:-$PKGS_LIST} /tmp/
 COPY prepare-image.sh prepare-ipxe.sh configure-nonroot.sh /bin/
 
+# some cachito magic
+COPY $REMOTE_SOURCES $REMOTE_SOURCES_DIR
```

Comment:
Does this do `COPY . .` by default?

Reply:
not exactly, it's limited to remote sources defined in cachito configuration, but it is actually managed by OSBS
REMOTE_SOURCES contains extracted tar archives provided by cachito in the buildroot workdir, one for each remote source
REMOTE_SOURCES_DIR in the build image contains a subdirectory for each remote source
Also $REMOTE_SOURCES_DIR/{name}/cachito.env (where {name} refers to the name of a given remote source) is a bash script with exported environment variables received from each cachito request, like PIP_CERT and PIP_INDEX_URL for pip, which is sourced later to be able to access the nexus repository
you can find more info here https://osbs.readthedocs.io/en/latest/users.html#remote-sources

Reply:
I guess my question boils down to: what happens if I just build this locally without providing these variables?

Reply:
local builds work just fine, it is the first thing I tested
did you encounter any issue with local builds?